src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -487,6 +487,7 @@ private function registerFragmentsConfiguration(array $config, ContainerBuilder
 
             return;
         }
+        $container->setParameter('fragment.renderer.hinclude.global_template', $config['hinclude_default_template']);
```

Comment:
Does it work even when providing the value via the old key and not via the new one?

Reply:
@Simperfit You are setting a parameter with no prior existence/emptiness check at 2 different places so only one can win (and potentially not the expected one). We need a test I guess :)